lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -84,7 +84,7 @@ def acts_as_list_top
             end
 
             def acts_as_list_class
-              self.class
+              ::#{self.name}
```

Comment:
Yes this is very helpful! I've missed that it happens inside the string!
Thank you!